ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/PhotoEdit.java

```diff
@@ -1,88 +1,70 @@
 package net.gini.android.vision.internal.camera.photo;
 
-import android.graphics.Bitmap;
-import android.graphics.BitmapFactory;
 import android.os.AsyncTask;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
-import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * @exclude
  */
 public class PhotoEdit {
 
-    private static final int DEF_QUALITY = 100;
-
     private final Photo mPhoto;
-    private int mRotationDegrees;
-    private int mQuality = DEF_QUALITY;
+    private final List<PhotoModifier> mPhotoModifiers;
 
-    public PhotoEdit(@NonNull Photo photo) {
+    PhotoEdit(@NonNull Photo photo) {
         mPhoto = photo;
-        resetToDefaults();
+        mPhotoModifiers = new ArrayList<>();
     }
 
     @NonNull
-    public PhotoEdit rotate(int degrees) {
-        degrees %= 360;
-        mRotationDegrees = degrees;
+    public PhotoEdit rotateTo(int degrees) {
```

Comment:
Clarified the intent to show that the image is rotated to the given degrees. So if it was at 90 and you call rotateTo(180) the image won't be orientated to 270 degrees but 180.